### ISSUE TITLE
cohttp 0.19.2 and async 113.00.00

### ISSUE THREAD

**Issue by artemkin** [mirage/ocaml-cohttp#429]
I've re-installed `4.02.3` and `113.00.00` packages from the scratch, but still can't install `cohttp` without downgrading. Any ideas?

```
$ opam install cohttp
The following actions will be performed:
  ↘  downgrade pipebang           113.00.00 to 110.01.00  [required by async_extra]
  ↘  downgrade type_conv          113.00.00 to 112.01.02  [required by bin_prot]
  ↘  downgrade sexplib            113.00.00 to 112.35.00  [required by cohttp]
  ↘  downgrade pa_ounit           113.00.00 to 112.35.00  [required by async_extra]
  ↘  downgrade fieldslib          113.00.00 to 109.20.03  [required by cohttp]
  ↘  downgrade comparelib         113.00.00 to 109.60.00  [uses type_conv]
  ↘  downgrade bin_prot           113.00.00 to 112.35.00  [required by async_extra]
  ↻  recompile variantslib        109.15.03               [uses type_conv]
  ↻  recompile enumerate          111.08.00               [uses type_conv]
  ↘  downgrade pa_structural_sexp 113.00.00 to 112.35.00  [uses type_conv]
  ↻  recompile uri                1.9.1                   [uses type_conv]
  ↻  recompile ipaddr             2.6.1                   [uses type_conv]
  ↘  downgrade pa_bench           113.00.00 to 112.06.00  [uses type_conv]
  ↘  downgrade custom_printf      113.00.00 to 112.24.00  [uses type_conv]
  ↻  recompile pa_test            112.24.00               [uses type_conv]
  ↘  downgrade typerep            113.00.00 to 112.35.00  [uses bin_prot]
  ↘  downgrade core_kernel        113.00.00 to 112.35.00  [required by async_kernel]
  ↘  downgrade core               113.00.00 to 112.35.01  [required by async_extra]
  ↘  downgrade async_kernel       113.00.00 to 112.35.00  [required by async]
  ↘  downgrade async_unix         113.00.00 to 112.35.00  [required by async]
  ↘  downgrade async_rpc_kernel   113.00.00 to 112.35.00  [required by async_extra]
  ↘  downgrade async_extra        113.00.00 to 112.35.00  [required by async]
  ↘  downgrade async              113.00.00 to 112.35.00  [required by cohttp]
  ↻  recompile cstruct            1.7.0                   [uses async]
  ↻  recompile xenstore           1.2.5                   [uses cstruct]
  ↻  recompile io-page            1.5.1                   [uses cstruct]
  ↻  recompile mirage-types       2.6.0                   [uses cstruct]
  ↻  recompile mirage-profile     0.5                     [uses cstruct]
  ↻  recompile mirage-types-lwt   2.6.0                   [uses cstruct]
  ↻  recompile mirage-clock-xen   1.0.0                   [uses mirage-types]
  ↻  recompile mirage-clock-unix  1.0.0                   [uses mirage-types]
  ↻  recompile xen-gnt            2.2.0                   [uses cstruct]
  ↻  recompile shared-memory-ring 1.1.1                   [uses cstruct]
  ↻  recompile mirage             2.6.1                   [uses ipaddr]
  ↻  recompile mirage-xen         2.3.3                   [uses cstruct]
  ↻  recompile mirage-unix        2.3.1                   [uses cstruct]
  ↻  recompile mirage-entropy-xen 0.3.0                   [uses cstruct]
  ↻  recompile ctypes             0.4.1                   [uses mirage-xen]
  ↻  recompile mirage-console     2.1.3                   [uses xenstore]
  ↘  downgrade async_ssl          113.00.00 to 112.35.00  [uses async]
  ↻  recompile conduit            0.8.8                   [uses async]
  ∗  install   cohttp             0.19.2                
===== ∗  1   ↻  22   ↘  19 =====
Do you want to continue ? [Y/n] 
```

**Comment by artemkin** [comment]
It seems fixed in 0.19.3.

**Comment by avsm** [comment]
Yes, it now supports latest Async.  Sorry for forgetting to update this issue!
